src/providers/windshaft.js

```diff
@@ -1,13 +1,76 @@
 var d3 = require("d3");
+var _ = require("underscore");
+var Providers = require("./");
 
 function WindshaftProvider(options) {
-	this.sql_api_template = options.sql_api_template || 'http://{user}.cartodb.com';
+	this.tiler_template = options.tiler_template || 'http://{user}.cartodb.com';
 	this.user = options.user;
 	this.table = options.table;
 	this.format = options.format;
+	this.options = options;
 	this.tileCache = {};
+	this.initialize();
 }
 
 WindshaftProvider.prototype = {
+	initialize: function(){
+		var self = this;
+		this.tiler_template = this.tiler_template.replace("{user}", this.user);
+		var mapconfig = this._generateMapconfig(this.table);
+		var url = this.tiler_template + "/api/v1/map?config=" + encodeURIComponent(JSON.stringify(mapconfig));
+		cartodb.d3.net.jsonp(url + "&callback=mapconfig", function(data){
+			self.layergroup = data;
```

Comment:
Yes sorry, sublime does this for some reason. It drives me crazy too. Correcting...